github/client.go

```diff
@@ -420,50 +420,79 @@ func (client *Client) ForkRepository(project *Project) (repo *octokit.Repository
 	return
 }
 
-func (client *Client) Issues(project *Project) (issues []octokit.Issue, err error) {
-	url, err := octokit.RepoIssuesURL.Expand(octokit.M{"owner": project.Owner, "repo": project.Name})
+type IssueParams struct {
```

Comment:
These structs map very closely to the JSON representation of these resources as returned from GH API. I'm open to document them better, but I'm not sure how. Do you have an idea of what you would like to see here?